README.md

```diff
@@ -27,9 +27,15 @@ The `Node` is started with a `prnm.Config` which contains all needed configurati
 After importing the `android/` folder in Android Studio, run it in the Emulator or on a real phone.  
 The opposite party can be either also an App, or a [perun-eth-demo](https://github.com/perun-network/perun-eth-demo)-node.
 
+## Features
+The current version of the bindings do not expose the full complexity of `go-perun` to the developer.  
+This is manly done to increase the development speed of simple demonstrator apps. Future releases will support all major features of `go-perun` and be suited for advanced projects.  
+
+The most interesting feature right now are `two-party single-asset ledger payment channels`.
+
```

Comment:
We already mention at the top
> Right now it only provides two-party-payment channels.

I'd delete this Feature section and just change the above statement to
> Right now, it supports two-party single-asset ledger payment channels.